imgui-SFML.cpp

```diff
@@ -876,11 +876,12 @@ void updateJoystickLStickState(ImGuiIO& io) {
 }
 
 void setClipboardText(void* /*userData*/, const char* text) {
-    sf::Clipboard::setString(text);
+    sf::Clipboard::setString(sf::String::fromUtf8(text, text + std::strlen(text)));
 }
 
 const char* getClipboadText(void* /*userData*/) {
-    s_clipboardText = sf::Clipboard::getString().toAnsiString();
+    std::basic_string<unsigned char> tmp = sf::Clipboard::getString().toUtf8();
+    s_clipboardText = std::string(tmp.cbegin(), tmp.cend());
```

Comment:
cbegin/cend is C++11 feature. We try to be C++03 compilable, so please use `begin` and `end` here